src/BlazorWebView/src/WindowsForms/Microsoft.AspNetCore.Components.WebView.WindowsForms.csproj

```diff
@@ -16,7 +16,7 @@
     <!-- NuGet package information -->
     <IsPackable>true</IsPackable>
     <PackageId>Microsoft.AspNetCore.Components.WebView.WindowsForms</PackageId>
-    <PackageIconFile>$(MauiRootDirectory)Assets\aspnet-icon.png</PackageIconFile>
+    <PackageIcon>$(MauiRootDirectory)Assets\aspnet-icon.png</PackageIcon>
```

Comment:
Why was this changed?

Reply:
Yeah this is because the arcade sdk also sets that property, setting PackageIcon, it reads this Prop and uses that, but I will confirm the packages have the correct icon.

Reply:
Good catch @mattleibow  seems something got wrong with the icon, do you have a way to compare 2 nupkg ? like one from main agains one from the PR see where they diff? 
[Microsoft.Maui.Controls.8.0.40-ci.net8.27947.zip](https://github.com/dotnet/maui/files/15083161/Microsoft.Maui.Controls.8.0.40-ci.net8.27947.zip)


<img width="1119" alt="Screenshot 2024-04-23 at 22 16 44" src="https://github.com/dotnet/maui/assets/1235097/d23b9ff2-b15a-4fe4-a281-1e93ac60b152">

Reply:
<img width="1155" alt="Screenshot 2024-04-24 at 12 52 49" src="https://github.com/dotnet/maui/assets/1235097/79d001dd-dddb-446d-9cd6-ceeff8c8c419">

Just confirmed with nupkg from this PR